### DISCUSSION TITLE
Vehicle too old - Please add C4 Grand Picasso, different VRN range

### DISCUSSION THREAD

**Discussion by cyrano1966** [flobz/psa_car_controller#817] [Q&A]
- !CarModel
  name: C4 Grand Picasso (2016)
  battery_power: 0
  fuel_capacity: 56
  abrp_name:
  reg: VF73ABHZMG.*

**Comment by vincentducate** [comment]
I'm having the same problem with a brand new Peugeot 308 Sw GT Hybrid 180
